src/gui/MainWindow.cpp

```diff
@@ -552,13 +555,20 @@ void MainWindow::finalize()
 								m_toolBar );
 	controllers_window->setShortcut( Qt::Key_F11 );
 
+    m_countDownClock = new LcdSpinBox(2, this, tr("Count Down Clock"));
+
+    m_countDownClock->setLabel( tr("SEC") );
```

Comment:
Well, in this case, we may want to implement a conversion algorithm and makes it public in `Engine` or  somewhere, this problem was also a headache in my previous PR (Go to feature), it will introduce offsets when doing the conversions.